### ISSUE TITLE
Windows serwer crash on main page youtube.com - cookie request

### ISSUE THREAD

**Issue by misiek998** [ttalvitie/browservice#117]
Tested serwer windows64 version 0.9.9.0 at youtube.com got cookie information window, anything I press makes serwer crash
Here is console window log 

`INFO @ retrojsvice.dll C:\build\src\viceplugins\retrojsvice\src\context.cpp:296 -- Creating retrojsvice plugin context
Fontconfig error: Cannot load default config file: No such file: (null)
[0305/201320.754:WARNING:resource_util.cc(94)] Please customize CefSettings.root_cache_path for your application. Use of the default value may lead to unintended process singleton behavior.
INFO @ retrojsvice.dll C:\build\src\viceplugins\retrojsvice\src\context.cpp:354 -- Starting plugin
INFO @ retrojsvice.dll C:\build\src\viceplugins\retrojsvice\src\http.cpp:620 -- Starting HTTP server (listen address: 127.0.0.1:8080)
INFO @ retrojsvice.dll C:\build\src\viceplugins\retrojsvice\src\http.cpp:628 -- HTTP server started successfully
INFO @ retrojsvice.dll C:\build\src\viceplugins\retrojsvice\src\window_manager.cpp:319 -- New window requested by user
INFO @ C:\build\src\src\server.cpp:50 -- Got request for new window from vice plugin
INFO @ C:\build\src\src\window.cpp:597 -- Creating window 1
INFO @ retrojsvice.dll C:\build\src\viceplugins\retrojsvice\src\window_manager.cpp:330 -- Creating window 1
INFO @ C:\build\src\src\window.cpp:207 -- CEF browser for window 1 created
INFO @ C:\build\src\src\bookmarks.cpp:134 -- Bookmark file 'C:\Users\Michas\AppData\Roaming\Browservice\bookmarks' does not exist, using empty set of bookmarks
[0305/201332.086:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738373 [8,5]
[0305/201332.095:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738373 [8,5]
[0305/201332.095:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738373 [8,5]
[0305/201332.095:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738373 [8,5]
[0305/201332.383:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738373 [8,5]
[0305/201333.709:INFO:CONSOLE(5129)] "LegacyDataMixin will be applied to all legacy elements.
Set `_legacyUndefinedCheck: true` on element class to enable.", source: https://www.youtube.com/s/desktop/3c5cb7b3/jsbin/desktop_polymer.vflset/desktop_polymer.js (5129)
[0305/201335.474:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738380 [8,12]
[0305/201335.719:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738380 [8,12]
[0305/201335.719:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738380 [8,12]
[0305/201335.722:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738380 [8,12]
[0305/201335.722:WARNING:browser_info.cc(319)] Returning a speculative frame for 34359738380 [8,12]
[0305/201337.180:INFO:CONSOLE(5129)] "LegacyDataMixin will be applied to all legacy elements.
Set `_legacyUndefinedCheck: true` on element class to enable.", source: https://www.youtube.com/s/desktop/3c5cb7b3/jsbin/desktop_polymer.vflset/desktop_polymer.js (5129)
[0305/201338.915:WARNING:obfuscated_file_util.cc(1336)] Failed to get origin+type directory: { uri: filesystem:https://www.youtube.com/temporary/, storage key: { origin: https://www.youtube.com, top-level site: https://youtube.com, nonce: <null>, ancestor chain bit: Same-Site }, bucket id: 1 } error:-4`

Can't move next to any video, serwer crash immediately.

**Comment by roschlynnmichael** [comment]
Yeah this has been happening to me as well. Still trying to figure out what is wrong and why its just youtube crashing. all other sites work great. 
Been using this to browse the web on a surface RT 2 using the IE Browser.
